src/NuGet.Core/NuGet.Packaging/Core/PackageDependencyGroup.cs

```diff
@@ -18,6 +19,21 @@ public class PackageDependencyGroup : IEquatable<PackageDependencyGroup>, IFrame
         private readonly NuGetFramework _targetFramework;
         private readonly IEnumerable<PackageDependency> _packages;
 
+        [JsonConstructor]
+        private PackageDependencyGroup(NuGetFramework targetFramework)
+        {
+            if (targetFramework == null)
+            {
+                _targetFramework = NuGetFramework.AnyFramework;
+            }
+            else
+            {
+                _targetFramework = targetFramework;
+            }
+
+            _packages = new List<PackageDependency>();
```

Comment:
how about `Enumerable.Empty<PackageDependency>()` instead of creating a new List that initializes an array with the default capacity?